backend/package-lock.json

```diff
@@ -3410,6 +3410,16 @@
       "resolved": "https://registry.npmjs.org/lodash.clonedeep/-/lodash.clonedeep-4.5.0.tgz",
       "integrity": "sha1-4j8/nE+Pvd6HJSnBBxhXoIblzO8="
     },
+    "lodash.foreach": {
```

Comment:
Is this a dependancy of mongose-unique-validator? Or do we not need to update the package-lock.json? 

@robinv85 any thoughts?

Reply:
You'll see the requirements here: Every package referenced as a `require` by packages we install for direct use in our app will then also be at top level of the package-lock.json.
```
    "mongoose-unique-validator": {
      "version": "2.0.3",
      "resolved": "https://registry.npmjs.org/mongoose-unique-validator/-/mongoose-unique-validator-2.0.3.tgz",
      "integrity": "sha512-3/8pmvAC1acBZS6eWKAWQUiZBlARE1wyWtjga4iQ2wDJeOfRlIKmAvTNHSZXKaAf7RCRUd7wh7as6yWAOrjpQg==",
      "requires": {
        "lodash.foreach": "^4.1.0",
        "lodash.get": "^4.0.2"
      }
    },
```

Reply:
Cool! Missed that part haha Thank you so much!